src/platform/datapath_winuser.c

```diff
@@ -1997,12 +2009,13 @@ QUIC_DISABLED_BY_FUZZER_START;
                 goto Error;
             }
 
+            #pragma warning(suppress:6387) // _Param_(8)' could be '0' - by design.
```

Comment:
Looks like we need to update SAL annotation for `RIOCreateRequestQueue`?

Reply:
Agreed, I filed a bug.